man/setkey.Rd

```diff
@@ -139,5 +143,14 @@ setindex(DT, A)
 setindex(DT, B)
 indices(DT)           # get indices single vector
 indices(DT, vectors = TRUE) # get indices list
+
+# Example illustrating the importance of using the dot (`.`) syntax with integer keys:
+DT = data.table(id = 2:1)
+setkey(DT, id)
+subset_value <- 1
+DT[subset_value]  # Incorrect: treats `subset_value` as an row number
+# To correctly subset based on the key column `id`, use the dot (`.`) syntax:
+DT[.(subset_value)]  # Correct: matches `subset_value` against `id`
```

Comment:
please remove correct/incorrect

Reply:
please remove the words correct and incorrect from comments, because they could be confusing.
As I see it both lines of codes are correct but for different purposes (select by row number or by matching to key)

Reply:
thanks for clarification, done 👍